.gitleaks.toml

```diff
@@ -12,6 +12,7 @@
         '''testsuite\/integration\/elytron-oidc-client\/src\/test\/java\/org\/wildfly\/test\/integration\/elytron\/oidc\/client''',
         '''testsuite\/integration\/manualmode\/src\/test\/java\/org\/jboss\/as\/test\/manualmode\/ejb\/ssl''',
         '''testsuite\/integration\/manualmode\/src\/test\/java\/org\/jboss\/as\/test\/manualmode\/ejb\/client\/outbound\/connection\/security''',
+        '''testsuite\/integration\/manualmode\/src\/test\/java\/org\/wildfly\/test\/manual\/elytron\/oidc\/client''',

```

Comment:
Looks like `client` can be removed here since the file is in the `testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/elytron/oidc/` directory:

https://github.com/wildfly/wildfly/blob/main/testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/elytron/oidc/KeycloakConfiguration.java

Reply:
Oops, fixed! 
Thanks